automationtestapp/src/main/java/com/microsoft/aad/automation/testapp/ResultActivity.java

```diff
@@ -61,6 +66,10 @@ protected void onCreate(Bundle savedInstanceState) {
         }
         mTextView.setText(resultText);
 
+        mLogView = (TextView) findViewById(R.id.adalLogs);
+        mLogView.setMovementMethod(new ScrollingMovementMethod());
```

Comment:
My understanding is that the textbox is used to save the logs, not for us to actually show it. IMHO, there is no need to set the movement method. :)